### ISSUE TITLE
[DASHBOARD] Static Variable Display Tab

### ISSUE THREAD

**Issue by dandelionmane** [tensorflow/tensorboard#53]
Migrated from https://github.com/tensorflow/tensorflow/issues/4714

> I do a lot of different runs which contain descriptions as well as different meta parameters. It would be nice to have a tab on TensorBoard so I can tell which experiment I am viewing.
> 
> My thought would be to simply add static variables from the Graph which might include Strings, Ints, Floats, (maybe matrix in table format if possible, like PCA parameters, maybe confusion matrix, etc). These may include a text description, current learning rate, network meta parameters, etc. It would be good to simple present the "current" value of these variables so if there is a search on Discount Rates for RL or variation such as learning rate, those can be viewed. This would be different from simply graphing the Learning rate as that shows the history rather than "current". I think of this as a TensorFlow "Dashboard" that I can setup.
> 
> Something like:
> tf.dashboard_summary(tags, tensor/value, collections=None, name=None)
> 
> eg:
> tf.dashboard_summary("Description:",description_string_tensor_or_python_string)
> tf.dashboard_summary("Learning Rate:",lr_tensor)
> tf.dashboard_summary("DiscountRate:",discountRate_python_variable)
> 
> In this case, a "DASHBOARD" tab on tensorflow would contain the 3 labels above and the tensor/value. If the value is a python variable, then it should be considered constant and will not change over the graph lifecycle. If it is a tensor/variable, then it should be pulled from the graph at each iteration.
> 
> I can help but not sure where to jump in to get this started.

**Comment by asanakoy** [comment]
Any updates on this feature?

**Comment by colaskirschoff** [comment]
Interested by this feature too, especially for hparams visualization !
Any update @dandelionmane ? Is there anything we can do ?

**Comment by chihuahua** [comment]
@dandelionmane filed #581.

Hence, adding native support for function parameters is a priority for TensorBoard. We plan to first make TensorBoard run on a SQL-based backend (spearheaded by @jart), which will make introducing structural features like this easier.

That being said, if this is high-priority enough, we can probably implement this feature using a config file today - we'd have a proto-based API that writes a pbtxt to disk.

**Comment by erichiller** [comment]
Currently I'm doing this (recording arbitrary run and model data) in Text Summary , which now that Markdown is working is a reasonable solution, but an automated inclusion of specific information would be useful as well.
